mixins/imageHelper.ts

```diff
@@ -9,87 +9,166 @@ import { BaseItemDto, ImageType } from '@jellyfin/client-axios';
 
 declare module '@nuxt/types' {
   interface Context {
-    getImageUrl: (id: string, type: string) => string;
     getImageUrlForElement: (
-      element: HTMLElement,
-      item: BaseItemDto,
-      type: ImageType
+      type: ImageType,
+      item?: BaseItemDto,
+      element?: HTMLElement,
+      tag?: string,
+      itemId?: string,
+      maxWidth?: number,
+      maxHeight?: number,
+      quality?: number,
+      limitByWidth?: boolean
+    ) => string;
```

Comment:
Probably better to pass this as an object

Reply:
How exactly I should expose this so suggestions appear in VSCode? I can't get that to work

Reply:
See comment below